### ISSUE TITLE
iOS VoiceOver cannot correctly read links embedded in text.

### ISSUE THREAD

**Issue by rayGuoRay** [archriss/react-native-render-html#690]
I'm trying to render the HTML <p>text<a>link</a>text</p> using RenderHtml. But on iOS devices with VoiceOver, the "link" isn't correctly announced as a clickable element. How can I ensure VoiceOver properly reads the accessibility content for the link?

the React Native code:

<RenderHtml
       contentWidth={width}
       source={{
           html: '<p>This topic provides links for <a href="battery">battery</a> and <a href="power">power</a> issues.</p>',
       }}
       tagsStyles={tagsStyles}
      {...htmlConfig}
/>

react-native-render-html version is 6.3.4

**Comment by jsamr** [comment]
Closing since this repository is deprecated. Please use `@native-html/render` instead, and report any issue to the official repo here: https://github.com/native-html/render
